pkg/util/pki/temporaryCertificate.go

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
suggestion: I don't think we use camelcase for filenames anywhere else? Would it be better keep to our current style of either using snake_case (`temporary_certificate.go`) just making it all one word (`temporarycertificate.go`)?